dali/operators/reader/loader/video/nvdecode/ColorSpace.cu

```diff
@@ -0,0 +1,83 @@
+/*
+* Copyright 2017-2021 NVIDIA Corporation.  All rights reserved.
+*
+* Please refer to the NVIDIA end user license agreement (EULA) associated
+* with this source code for terms and conditions that govern your use of
+* this software. Any use, reproduction, disclosure, or distribution of
+* this software and related documentation outside the terms of the EULA
+* is strictly prohibited.
+*
+*/
+
+#include "ColorSpace.h"
+
+typedef struct {
+    uint8_t r, g, b;
+} Rgb;
+
+
+__constant__ float matYuv2Rgb[3][3] = {
+    1.164383f,  0.0f,       1.596027f,
+    1.164383f, -0.391762f, -0.812968f,
+    1.164383f,  2.017232f,  0.0f
+};
+
+__device__ static uint8_t Clamp(float x, float lower, float upper) {
+    return x < lower ? lower : (x > upper ? upper : x);

```

Comment:
Can you check imgproc.cu?
Should `return fminf(fmaxf(x, 0.0f), max);` be faster?

Reply:
Not sure, but definitely it is cooler.

Done